open_event/static/admin/css/material-custom.css

```diff
@@ -7840,3 +7840,6 @@ hr.on-light {
 .shadow-z-5 {
   box-shadow: 0 27px 24px 0 rgba(0, 0, 0, 0.2), 0 40px 77px 0 rgba(0, 0, 0, 0.22);
 }
+.auth_required {
+  margin-bottom: -15px;
+}
```

Comment:
@PiotrKowalski Please don't modify libraries css, You can add changes to file open-event.css